resources/Microsoft.Windows.Setting.WindowsUpdate/Microsoft.Windows.Setting.WindowsUpdate.psm1

```diff
@@ -0,0 +1,377 @@
+$global:WindowsUpdateSettingPath = 'HKLM:\SOFTWARE\Microsoft\WindowsUpdate\UX\Settings'
```

Comment:
These should be initialized like the accessibility module so the actual registry isn't touched during the tests

Reply:
Both HKLM and HKU are not supported in Pester :(

Reply:
Are you sure? It looks like it was working for the UAC control in #42 - https://github.com/microsoft/winget-dsc/blob/61d2ebf668659f0d3725e171d1c5db8ecbbe93be/resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1#L152-L160